.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+<!---
+Verify first that your issue/request is not already reported on GitHub.
```

Comment:
Maybe start this sentence with "Please..."? :)

Reply:
Done.